jwt/README.md

```diff
@@ -0,0 +1,79 @@
+# `jwt` <!-- omit in toc -->
+
+# Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Usage](#usage)
+  - [Signing](#signing)
+  - [Verifying](#verifying)
+- [Directory Structure](#directory-structure)
+    - [Rationale](#rationale)
+
+
+# Usage 
+
+## Signing
+
+```go
+package main
```

Comment:
FYI you can include [examples](https://go.dev/blog/examples) in tests, which show up in the godocs.